stories/__tests__/SQFormAsyncAutocomplete.stories.test.js

```diff
@@ -0,0 +1,185 @@
+import React from 'react';
+import {composeStories} from '@storybook/testing-react';
+import {render, screen, within} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import * as stories from '../SQFormAsyncAutocomplete.stories';
+
+const {
+  Default: SQFormAsyncAutocomplete,
+  WithValidation: SQFormAsyncAutocompleteWithValidation
+} = composeStories(stories);
+
+describe('SQFormAsyncAutocomplete Tests', () => {
+  describe('AsyncAutocomplete Only', () => {
+    it('should render a dropdown, title, and 2 buttons', () => {
```

Comment:
This test description mentions two buttons but I'm only seeing one button being looked for in the test.

Reply:
Good catch! I removed one of them in my last update, but didn't update the test name.